web/components/reports/FcReportMetadata.vue

```diff
@@ -32,20 +49,86 @@
 </template>
 
 <script>
+import { mapGetters } from 'vuex';
+import { ReportType } from '@/lib/Constants';
 import FcTextReportValue from '@/web/components/data/FcTextReportValue.vue';
+import FcButton from '@/web/components/inputs/FcButton.vue';
 
 export default {
   name: 'FcReportMetadata',
   components: {
+    FcButton,
     FcTextReportValue,
   },
   props: {
     entries: Array,
+    type: ReportType,
+  },
+  data() {
+    return {
+      isExpanded: false,
+    };
   },
   computed: {
     showCallOut() {
       return this.$parent.type && this.$parent.type.name === 'COUNT_SUMMARY_TURNING_MOVEMENT';
     },
+    collisionFilters() {
+      const collisionFilters = [...this.filterChipsCollision()];
+      const daysOfWeek = this.filterChipsCommon().filter(item => item.filter === 'daysOfWeek')[0];
+      if (daysOfWeek) {
+        collisionFilters.push(daysOfWeek);
+      }
+      return this.readableFilters(collisionFilters);
+    },
+    numFilters() {
+      return this.collisionFilters.length;
+    },
+  },
+  methods: {
+    readableFilters(filters) {
+      const readableFilters = [];
+      for (let i = 0; i < filters.length; i++) {
+        const { filter, label } = filters[i];
+        let newFilter = '';
+
+        if (filter === 'validated') {
```

Comment:
i'd do something like
```js
let mapping = {
  daysOfWeek : 'Days of the week', 
  ....
}
newFilter = mapping[filter] || ''
```

Reply:
Are these available to import from lib/Constants.js? 
... if not, maybe a switch statement would organize it a bit more.